cura/OAuth2/KeyringAttribute.py

```diff
@@ -18,6 +18,9 @@
     import win32timezone
     from keyring.backends.Windows import WinVaultKeyring
     keyring.set_keyring(WinVaultKeyring())
+if Platform.isOSX() and hasattr(sys, "frozen"):
+    from keyring.backends.OS_X import Keyring
```

Comment:
According to https://keyring.readthedocs.io/en/latest/history.html#v22-0-0 the OS_X is depreceted since v22.0.0 and we should use macOS instead (to avoid the deprecation warning).

```suggestion
    from keyring.backends.macOS import Keyring
```